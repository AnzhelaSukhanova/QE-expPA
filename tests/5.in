(set-option :produce-models true)
(set-logic BV)
(declare-fun y () (_ BitVec 4))
(declare-fun z () (_ BitVec 4))
(assert
 (exists ((x (_ BitVec 4))) (and 
 (or (bvult x y) (= x y)) 
 (or (bvult z x) (= z x)))))
