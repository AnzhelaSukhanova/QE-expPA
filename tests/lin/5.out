(not (bvult (bvmul y #b1100) (bvmul y #b0011)))
